blockchain/My-Wallet-V3-iOS
# feat(Announcement Cards): add preliminary support for new announcement cards (IOS-1235)

## Objective

Add the _Continue Verification_ announcement card, which allows users to resume the on-boarding process.

## Description

Added preliminary support for the `AnnouncementCard` object.

## How to Test

Launch KYC, select **Begin Now**, close KYC, select a different tab in the tab bar controller, select the Dashboard tab once again and notice the announcement card.

## Screenshot/Design assessment

![simulator screen shot - iphone x - 2018-09-18 at 08 17 53](https://user-images.githubusercontent.com/14079155/45686964-766a7700-bb1b-11e8-97ee-1becaacea4a0.png)

## Merge Checklist
